certbot/certbot/plugins/enhancements.py

```diff
@@ -140,7 +156,8 @@ def deploy_autohsts(self, lineage, *args, **kwargs):
         """
 
     @abc.abstractmethod
-    def enable_autohsts(self, lineage, domains, *args, **kwargs):
+    def enable_autohsts(self, lineage: interfaces.RenewableCert, domains: Iterable[str],
```

Comment:
Based on how `enhancements.enable` is typed, it seems likely that `lineage` here would end up being `Optional`.

Reply:
Fixed that.